src/pages/Age/index.jsx

```diff
@@ -1,33 +1,33 @@
 import React from "react";
 import { useHistory } from 'react-router-dom';
+import { Container, Row, Col, Button } from 'reactstrap';
 
 import "./style.scss";
 
 const Age = props => {
   const history = useHistory();
 
   return (
-    <div className="AgeContainer">
-      <main className="AgeWrapper">
-        <div className="AgeContent">
-          <section>
-            <p style={{
-                textAlign: 'center',
-                fontFamily: 'Permanent Marker',
-                fontSize: 36,
-              }}
-            >
-              Qual a sua idade?
-            </p>
+    <Container fluid className="AgeWrapper">
```

Comment:
i suggest we use bam here to keep it an easy pattern
.Age {
&.-wrapper {}
&-Content{}
}